Makefile

```diff
@@ -340,6 +344,32 @@ e2e-cosign-setup:
 	./cosign-linux-amd64 sign --allow-insecure-registry --allow-http-registry --tlog-upload=false --key cosign.key ${TEST_REGISTRY}/cosign@`${GITHUB_WORKSPACE}/bin/oras manifest fetch ${TEST_REGISTRY}/cosign:signed-key --descriptor | jq .digest | xargs` && \
 	./cosign-linux-amd64 sign --allow-insecure-registry --allow-http-registry --tlog-upload=false --key cosign.key ${TEST_REGISTRY}/all@`${GITHUB_WORKSPACE}/bin/oras manifest fetch ${TEST_REGISTRY}/all:v0 --descriptor | jq .digest | xargs`
 
+e2e-cosign-akv-setup:
+	rm -rf .staging/cosign
+	mkdir -p .staging/cosign
+	curl -sSLO https://github.com/sigstore/cosign/releases/download/v${COSIGN_VERSION}/cosign-linux-amd64
```

Comment:
was just looking at cosign version, it looks like they continue to release v1.1.x ,  whats the big differ in v1.x.x vs v2.x.x?